components/handlers/module_overview.py

```diff
@@ -104,7 +104,7 @@ def GET(self):
         system_message = SESSION['editModMsg']
         SESSION['editModMsg'] = None
 
-        return RENDER.viewModule_updated(module_info, self.fixed_mounting_plan,
+        return RENDER.viewModule(module_info, self.fixed_mounting_plan,
                                          self.tenta_mounting_plan, number_of_student_planning,
                                          system_message)
 
```

Comment:
Line 96: trailing whitespace
Lines 108,109: wrong continued indentation
This page is not pylinted.